Gemfile

```diff
@@ -36,14 +37,12 @@ group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
   gem 'rspec-rails', '~> 5.0.0'
-  gem 'email_validator', '~> 2.2', '>= 2.2.3'
-  gem 'phonelib', '~> 0.5.4'
```

Comment:
do you need to remove it? I thought someone is using it

Reply:
These gems are already in the development section with the same version.